pkg/config/components/OTelDebugExporter.yaml

```diff
@@ -0,0 +1,27 @@
+name: otel_debug_exporter
+kind: OTelDebugExporter
+summary: Exports pipeline signal traffic to stdout
+description: |
+  Exports signal data in a pipeline to stdout.
+properties:
+  - name: Verbosity
```

Comment:
How much of the surface of the debug exporter config would we like to surface in this initial component version?

Reply:
I think only surfacing `Verbosity` is good for now